### PR TITLE
Use new features extensions

### PR DESCRIPTION
**Pull request by xxbillouxx**
Depends-On: https://github.com/wazo-platform/xivo-dao/pull/239

### CODE REVIEW

**Comment by xxbillouxx** [comment]
recheck

**Comment by xxbillouxx** [comment]
recheck

**Comment by xxbillouxx** [comment]
recheck

**Comment by xxbillouxx** [comment]
recheck

**Comment by xxbillouxx** [comment]
recheck

**Comment by xxbillouxx** [comment]
recheck

**Comment by xxbillouxx** [comment]
recheck

**Comment by pc-m** [comment]
recheck

**Comment by pc-m** [comment]
recheck

File: integration_tests/suite/helpers/helpers/extension_feature.py [hunk 1]
```diff
@@ -14,18 +14,18 @@ def generate_extension_feature(**parameters):
 
 def add_extension_feature(**parameters):
     with db.queries() as queries:
-        id = queries.insert_extension_feature(**parameters)
+        uuid = queries.insert_extension_feature(**parameters)
     return {
-        'id': id,
+        'uuid': str(uuid),
         'exten': parameters.get('exten'),
         'context': 'xivo-features',
```
Comment (pc-m) [inline integration_tests/suite/helpers/helpers/extension_feature.py:21]: Does it still make sense to return the context `xivo-features` here?

File: wazo_confd/plugins/extension_feature/api.yml [hunk 2]
```diff
@@ -17,15 +17,15 @@ paths:
           description: Extension feature list
           schema:
             $ref: '#/definitions/ExtensionFeatureItems'
-  /extensions/features/{extension_id}:
+  /extensions/features/{extension_uuid}:
```
Comment (pc-m) [inline wazo_confd/plugins/extension_feature/api.yml:20]: That's an API change. Does portal or wazo-ui use `/extensions/features/<id>`?

File: wazo_confd/plugins/extension_feature/service.py [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
+from xivo_dao.resources.feature_extension import dao as feature_extension_dao
 
 from wazo_confd.helpers.resource import CRUDService
 
 from .notifier import build_notifier
 from .validator import build_validator
 
 
-class ExtensionService(CRUDService):
+class FeatureExtensionService(CRUDService):
```
Comment (pc-m) [inline wazo_confd/plugins/extension_feature/service.py:13]: I think search and get can be removed from this class

**Review by pc-m — CHANGES_REQUESTED** [review]

File: wazo_confd/plugins/extension_feature/api.yml (hunk 2 above)
Comment (xxbillouxx) [reply wazo_confd/plugins/extension_feature/api.yml:20]: Internal ticket created : https://wazo-dev.atlassian.net/browse/NES-2163
And new PR created too: https://github.com/wazo-platform/wazo-ui/pull/149

File: wazo_confd/plugins/extension_feature/service.py (hunk 3 above)
Comment (xxbillouxx) [reply wazo_confd/plugins/extension_feature/service.py:13]: @pc-m get/search removed

**Review by pc-m — APPROVED** [review]

**Review by pc-m — APPROVED** [review]

**Comment by pc-m** [comment]
recheck

**Comment by pc-m** [comment]
regate

**Pull request merged**